tests/roles/linux-system-roles.postgresql/defaults

```diff
@@ -0,0 +1 @@
+../../../defaults
```

Comment:
For some reason these are not being reported in git as symlinks - they are reported as files with the contents like this:
```
../../../defaults
```
Try this:
```
cd tests/roles/linux-system-roles.postgresql
for dir in defaults handlers meta tasks templates vars ; do
  rm -rf "$dir"
  git rm -f "$dir"
  ln -s ../../../"$dir"
  git add "$dir"
done
```